main/authorization.bs

```diff
@@ -19,3 +21,31 @@ the HTTP `Link` header with a `rel` value of `acl` [[#acl]].
 In the event that a server can't apply an ACL to a resource, it MUST deny
 access.
 [[Source](https://github.com/solid/specification/issues/130#issue-532777017)]
+
+Servers exposing client's access privileges on a resource URL MUST advertise
+by including the `WAC-Allow` HTTP header in the response of HTTP `HEAD` and
```

Comment:
Don't we have an issue where `WAC-Allow` is under discussion?
I understand that this PR is just an officialized version of the draft spec, but as the author of the original text and library, I would like to reiterate my point from https://github.com/solid/specification/issues/170#issuecomment-523401269 that I am _not_ in favor of a custom header. I would rather see the `Link` header here.

Should we at least acknowledge in the spec that this point is under discussion?

Reply:
Either is fine with me. As mentioned, I wasn't crazy about the custom header either.

Will implement whatever comes out of it in dokieli eg. whether to disable the "save" button or not.

Reply:
As an editor, I have indeed noted what you've said ie. not in favour of custom header and that the Link header may be a preferable alternative. However, I didn't think that it was significantly under discussion considering that there was a draft spec and implementations in place. To make better editorial calls, we need more feedback about the experience re libraries and applications. What experience can you share from the implementations? Is there an issue/PR for an alternative approach or implemented somewhere?

If we can go through similar steps for an alternative, as we did with WAC-Allow, we can raise that as a possible replacement (eg. Link header in https://github.com/solid/specification/pull/210#issuecomment-728265065 ). Alternatively, have a survey and see if the existing WAC-Allow implementations are stale, broken, or not actually used by applications. That could also justify an alternative.

Reply:
> To make better editorial calls, we need more feedback about the experience re libraries and applications. What experience can you share from the implementations?

I think this is tricky.

Yes, the implementation works and it was easy.

But it was easy because we designed and implemented it in an afternoon.

My issue is rather with it not being sustainable as an extension mechanism; we shoudn't keep on minting custom headerss.

Reply:
Unless there is a strong objection, we can go with the proposal for the time being. We can always revisit this when there is more information that we can consider.